libs/sqlite3/Makefile

```diff
@@ -69,7 +69,9 @@ $(call Package/sqlite3/Default/description)
 endef
 
 TARGET_CFLAGS += $(FPIC) \
-	-DSQLITE_ENABLE_UNLOCK_NOTIFY \
+	-DSQLITE_ENABLE_UNLOCK_NOTIFY=1 \
+	-DHAVE_ISNAN=1 \
+	-DHAVE_MALLOC_USABLE_SIZE=1 \
```

Comment:
https://github.com/diizzyy/packages/blob/847ebdc9096f85dbba2f403608a73e21f0bd8042/libs/sqlite3/Makefile#L72 ? Is that also wrong? :-)